### ISSUE TITLE
Should not lint on adding data members to existing R6 class

### ISSUE THREAD

**Issue by radbasa** [Appsilon/box.linters#23]
## box.linters version

0.0.0.9006

## Source code to lint

```R
box::use(
  R6[R6Class],
)

SomeClass <- R6Class("SomeClass",   # nolint
  public = list()
)

SomeClass$set("public", "x", 10)

s <- SomeClass$new()
s$x
```

## Lint command

```R
lintr::lint(code, linters = lintr::linters_with_defaults(defaults = box.linters::rhino_default_linters))
```

## Lint result

```
<text>:10:1: warning: [box_usage_linter] <package/module>$function does not exist.
SomeClass$set("public", "x", 10)
^~~~~~~~~~~~~
```

## Previous issues

- #22 

## Expected result

- [x] Should not lint on public attribute call `s$x`.
- [ ] Should not lint on R6 `set` command
